app/src/main/java/com/example/kuba10/mypokemonplaces/AddPlaceFragment/AddPlaceFragment.java

```diff
@@ -1,169 +0,0 @@
-import com.example.kuba10.mypokemonplaces.Model.PokePlace;
-import com.example.kuba10.mypokemonplaces.Model.PokemonGo;
-import com.example.kuba10.mypokemonplaces.R;
-import com.google.android.gms.maps.model.LatLng;
-import com.squareup.picasso.Picasso;
-
-import java.util.ArrayList;
-import java.util.Date;
-
-import butterknife.BindView;
-import butterknife.ButterKnife;
-
-
-
-public class AddPlaceFragment extends Fragment {
-
-    private FragmentListener fragmentListener;
-    private ArrayList<PokemonGo> pokemonGo_data_list;
-    private PokemonGo selectedPokemon;
-
-    @BindView(R.id.pokemon_image_view)
-    ImageView pokemonImageView;
-    @BindView(R.id.description_input)
-    TextView descriptionField;
-    @BindView(R.id.title_input)
-    TextView titleField;
-    @BindView(R.id.add_button)
-    Button addButt;
-    @BindView(R.id.cancel_button)
-    Button cancelButt;
-
-    public static AddPlaceFragment newInstance(LatLng position) {
-        AddPlaceFragment fragment = new AddPlaceFragment();
-        Bundle args = new Bundle();
-        args.putParcelable(Constants.POSITION, position);
-        fragment.setArguments(args);
-        return fragment;
-    }
-
-
-    @Override
-    public void onAttach(Context context) {
-        super.onAttach(context);
-        fragmentListener = (FragmentListener) context;
-        pokemonGo_data_list = fragmentListener.getPokemonList();
-    }
-    @Override
-    public View onCreateView(LayoutInflater inflater, ViewGroup container,
-                             Bundle savedInstanceState) {
-        View view = inflater.inflate(R.layout.fragment_add_place, container, false);
-
-        ButterKnife.bind(this, view);
-
-        addButt.setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View view) {
-
-                LatLng position = getArguments().getParcelable(Constants.POSITION);
-                double lat = position.latitude;
-                double lng = position.longitude;
-
-                PokePlace place = new PokePlace();
-
-                if (titleField.getText().toString().equals("")) {
-                    fragmentListener.showSnackbar(getString(R.string.placeMustHaveTitle));
-                } else {
-
-                    place.setTitle(titleField.getText().toString());
-                    place.setTitle(titleField.getText().toString());
-                    place.setDesctription(descriptionField.getText().toString());
-                    place.setLong(lng);
-                    place.setLat(lat);
-                    place.setListPosition(Constants.INDEX_NOT_SET);
-                    place.setFavourite(0);
-
-                    setSelectedPokemonId(place);
-                    setPlaceId(place);
-
-                    fragmentListener.savePlace(place);
-
-                    dismiss();
-
-                }
-
-
-            }
-        });
-        cancelButt.setOnClickListener(new View.OnClickListener()
```

Comment:
onClickListenery do argumentów, a jeśli są takie małe jak tu to sugeruję dorzucić retrolambdę (albo zobaczyć jak włączyć natywne wsparcie dla java 8 features w androidzie: https://developer.android.com/studio/write/java8-support.html

Reply:
Zostawie to na następny projekt - w tym jest już dość komplikacji która wyrywa się spod kontroli...